packages/marketplace/src/components/ui/developer-invite-member-modal/developer-invite-member-modal.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+import { Modal, ModalProps, SubTitleH5, FlexContainerBasic, Form } from '@reapit/elements'
+import { Formik } from 'formik'
+import styles from '@/styles/blocks/developer-invite-member.scss?mod'
+import { FIELD_NAMES } from './constants'
+import { validate, handleSubmit } from './handlers'
+import DeveloperInviteMemberModalInput from './developer-invite-member-modal-input'
+import DeveloperInviteMemberModalFooter from './developer-invite-member-footer'
+
+export type DeveloperInviteMemberModalProps = Pick<ModalProps, 'afterClose'> & {
+  visible?: boolean
+}
+
+export const initialValues = {
+  [FIELD_NAMES.NAME]: '',
+  [FIELD_NAMES.EMAIL]: '',
+  [FIELD_NAMES.MESSAGE]: '',
+}
+
+export const DeveloperInviteMemberModal: React.FC<DeveloperInviteMemberModalProps> = ({
+  visible = false,
+  afterClose,
+}) => {
```

Comment:
I think should unmount modal to avoid initialValues weird behavior and it'll not render when we don't use it
```
if (!visible){
  return null
}
```